mev_inspect/crud/total_profits.py

```diff
@@ -0,0 +1,23 @@
+import json
+from typing import List
+
+from mev_inspect.db import write_as_csv
+from mev_inspect.schemas.total_profits import TotalProfits
+
+
+def write_total_profits_for_blocks(
+        db_session,
```

Comment:
- [ ] rename to `inspect_db_session`

Reply:
Done in [7aa4c16](https://github.com/IlluvatarEru/mev-inspect-py/pull/6/commits/7aa4c16f520e4bf32bd4d88602685118bd265891), along with formatting of the code.